index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href='http://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
```

Comment:
Try to be consistent with your use of ` />`. You don't have it on this `link` tag, but you do on the next line